src/main/java/tk/shanebee/hg/listeners/GameListener.java

```diff
@@ -447,6 +456,29 @@ private void onInteract(PlayerInteractEvent event) {
 		}
 	}
 
+	@EventHandler
+	private void onInventoryClick(InventoryClickEvent e){
+		if (e.getWhoClicked() instanceof Player){
+			Player player = (Player) e.getWhoClicked();
+			InventoryAction action = e.getAction(); //might not be needed
+			Game g1 = playerManager.getPlayerData(player).getGame();
```

Comment:
g1 should be something a little more descriptive. Off the top of my head im not sure if "game" is used within this class, but it would probably be best to use that, or maybe even "game1" if "game" is already used